lib/main/atom/tooltipManager.ts

```diff
@@ -74,6 +74,8 @@ export function attach(editorView: JQuery, editor: AtomCore.IEditor) {
         if (exprTypeTooltip) return;
 
         var pixelPt = pixelPositionFromMouseEvent(editorView, e);
+        pixelPt.top += editor.displayBuffer.getScrollTop();
+        pixelPt.left += editor.displayBuffer.getScrollLeft();
         var screenPt = editor.screenPositionForPixelPosition(pixelPt);
```

Comment:
Maybe move this logic *into* `pixelPositionFromMouseEvent`. I think just remove the -left/top here : https://github.com/TypeStrong/atom-typescript/blob/master/lib/main/atom/tooltipManager.ts#L140-L141